content/articles/files-and-exeptions-in-python/index.md

```diff
@@ -0,0 +1,265 @@
+ 
+
+Working with files will make your programs fast when analyzing masses of data. Exceptions are special objects that Python uses to manage errors that occur when a program is running. In this tutorial, we’ll learn working with files and handling some exception errors.
+
+### Prerequisites
+
+Have some basic knowledge of Python language.
+
+### Reading from a file
+
+Working with data in a text file, you need to read the file into memory first. To read a file, you must have a file with some data.
```

Comment:
`To read a file, you must have a file with some data.`

This is not true. You can read an empty file as well.